src/components/GrowlNotification/index.tsx

```diff
@@ -11,20 +12,21 @@ import useTheme from '@styles/themes/useTheme';
 import useThemeStyles from '@styles/useThemeStyles';
 import CONST from '@src/CONST';
 import GrowlNotificationContainer from './GrowlNotificationContainer';
+import {GrowlNotificationProps} from './types';
 
 const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
 
-function GrowlNotification(_, ref) {
+function GrowlNotification({ref}: GrowlNotificationProps) {
```

Comment:
It doesn't look right, have you tested it?

Reply:
Have a [read](https://react.dev/reference/react/forwardRef) @cdOut! It should look more like this:
```ts
function GrowlNotification(_: unknown, ref: ForwardedRef<GrowlRef>) {
```